### PR TITLE
Fix integration tests

### PR DESCRIPTION
**Pull request by fhoering**
- use safe_experiment_fn for keras experiments
- use pex 2.1.1 for runs with tf 1.15
- only use import lib remotely in safe_experiment_fn

Change-Id: I5b498ffefecb3ce4cde0703be9d16dfd974f4e11

### CODE REVIEW

File: examples/linear_classifier_example.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def eval_input_fn():
         n_classes=winequality.get_n_classes())
     return Experiment(
         estimator,
-        tf.estimator.TrainSpec(train_input_fn, max_steps=10),
+        tf.estimator.TrainSpec(train_input_fn, max_steps=100),
```
Comment (kowalskin) [inline examples/linear_classifier_example.py:61]: Why the change of max steps?

File: examples/mlflow_example.py [hunk 2]
```diff
@@ -67,10 +67,10 @@ def eval_input_fn():
         ))
     return Experiment(
         estimator,
-        tf.estimator.TrainSpec(train_input_fn, max_steps=10000),
+        tf.estimator.TrainSpec(train_input_fn, max_steps=100),
```
Comment (kowalskin) [inline examples/mlflow_example.py:70]: Why change of max steps

**Review by kowalskin — COMMENTED** [review]
Why the change of max steps in some examples? To uniformize them between examples?

File: examples/linear_classifier_example.py (hunk 1 above)
Comment (fhoering) [reply examples/linear_classifier_example.py:61]: I know it is not related to this review but I don't want to do another want. It is to align all tests with 100 steps. It is enough to get the code running but reasonable fast for just running the samples as integration tests.

File: examples/mlflow_example.py (hunk 2 above)
Comment (fhoering) [reply examples/mlflow_example.py:70]: I know it is not related to this review but I don't want to do another want. It is to align all tests with 100 steps. It is enough to get the code running but reasonable fast for just running the samples as integration tests.

**Review by kowalskin — APPROVED** [review]
